Dockerfile

```
@@ -1,10 +1,35 @@
+COPY . .
+
+RUN npx next build && \
+    npx next-export-optimize-images
+
+
+# ---------- Production Stage ----------
+FROM base AS production
+
+COPY --from=build /app/out ./out
+
+RUN npm install -g serve
```

Hadolint (2.12.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)